tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/jdbc/LoaderFactory.java

```diff
@@ -30,7 +30,7 @@
     Loader create(QueryExecutor queryExecutor, String tableName)
             throws SQLException
     {
-        List<JDBCType> columnTypes = queryExecutor.executeQuery("SELECT * FROM " + tableName + " LIMIT 1").getColumnTypes();
+        List<JDBCType> columnTypes = queryExecutor.executeQuery("SELECT * FROM " + tableName + " WHERE 1=2").getColumnTypes();
```

Comment:
Is `getColumnCount()` not working? I feel that this method is more appropriate (if it works), since it conveys directly what the code is doing.